janusgraph-cql/src/test/java/org/janusgraph/blueprints/cql/structure/CQLStructureTest.java

```diff
@@ -0,0 +1,33 @@
+// Copyright 2017 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.blueprints.cql.structure;
+
+import org.apache.tinkerpop.gremlin.GraphProviderClass;
+import org.apache.tinkerpop.gremlin.structure.StructureStandardSuite;
+import org.janusgraph.blueprints.cql.CQLGraphProvider;
+import org.janusgraph.core.JanusGraph;
+import org.junit.runner.RunWith;
+
+@RunWith(StructureStandardSuite.class)
+@GraphProviderClass(provider = CQLGraphProvider.class, graph = JanusGraph.class)
+public class CQLStructureTest {
+
+//    TP3 ignores @BeforeClass -- the following method is never executed
```

Comment:
maybe leave just the comment that TP3 ignores @BeforeClass and get rid of the commented out method